CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ generated correctly.
 
 ### Removed
 - Removed check subdue e2e test.
+- Removed deprecated import command.
```

Comment:
This should go under a `### Breaking Changes` header per the community guideliens which @portertech and @grepory have agreed the core will follow.

https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md